wgengine/monitor/monitor.go

```diff
@@ -104,8 +104,10 @@ func New(logf logger.Logf) (*Mon, error) {
 	return m, nil
 }
 
-// InterfaceState returns the state of the machine's network interfaces,
-// without any Tailscale ones.
+// InterfaceState returns the latest snapshot of the machine's network
```

Comment:
Nit: put the whole first sentence on the first line.

Reply:
That goes over 80 characters and my editor wraps comments before that. The rest of this file (e.g. the several funcs just below this) are consistent at wrapping before 80 too.

So I'll keep as is. There's at least a blank line before the second paragraph?